--- eccodes-1.3.2.p1/docs/conf.py	2021-04-17 10:55:56.098050726 +0200
+++ eccodes-1.3.2.p1p2/docs/conf.py	2021-04-17 10:56:30.645189523 +0200
@@ -32,7 +32,7 @@ source_suffix = ".rst"
 master_doc = "index"
 
 # General information about the project.
-project = u"eccodes-python"
+project = u"eccodes"
 copyright = u"2017-, European Centre for Medium-Range Weather Forecasts (ECMWF)."
 
 # The version info for the project you're documenting, acts as replacement
@@ -40,7 +40,7 @@ copyright = u"2017-, European Centre for
 # the built documents.
 #
 # The full version, including alpha/beta/rc tags.
-release = pkg_resources.get_distribution("eccodes-python").version
+release = pkg_resources.get_distribution("eccodes").version
 # The short X.Y version.
 version = ".".join(release.split(".")[:2])
 
