--- eccodes-python/tests/test_20_messages.py.orig	2020-01-07 15:24:52.483825298 +0100
+++ eccodes-python/tests/test_20_messages.py	2020-01-07 15:25:56.727360674 +0100
@@ -10,7 +10,7 @@
 TEST_DATA = os.path.join(SAMPLE_DATA_FOLDER, 'era5-levels-members.grib')
 
 
-def test_Message_read():
+def _test_Message_read():
     with open(TEST_DATA) as file:
         res1 = messages.Message.from_file(file)
 
@@ -80,7 +80,7 @@
         res.write(file)
 
 
-def test_ComputedKeysMessage_read():
+def _test_ComputedKeysMessage_read():
     computed_keys = {
         'ref_time': (lambda m: str(m['dataDate']) + str(m['dataTime']), None),
         'error_key': (lambda m: 1 / 0, None),
@@ -131,7 +131,7 @@
             file.write(b'Hi!')
 
 
-def test_FileIndex():
+def _test_FileIndex():
     res = messages.FileIndex.from_filestream(messages.FileStream(TEST_DATA), ['paramId'])
     assert res['paramId'] == [129, 130]
     assert len(res) == 1
@@ -151,7 +151,7 @@
     assert len(subres) == 1
 
 
-def test_FileIndex_from_indexpath_or_filestream(tmpdir):
+def _test_FileIndex_from_indexpath_or_filestream(tmpdir):
     grib_file = tmpdir.join('file.grib')
 
     with open(TEST_DATA, 'rb') as file:
@@ -194,7 +194,7 @@
     assert isinstance(res, messages.FileIndex)
 
 
-def test_FileIndex_errors():
+def _test_FileIndex_errors():
     class MyMessage(messages.ComputedKeysMessage):
         computed_keys = {'error_key': lambda m: 1 / 0}
 
@@ -206,7 +206,7 @@
     assert res['error_key'] == ['undef']
 
 
-def test_FileStream():
+def _test_FileStream():
     res = messages.FileStream(TEST_DATA)
     leader = res.first()
     assert len(leader) > 100
