--- eccodes-1.1.0.unchanged/setup.py	2020-10-15 17:56:17.000000000 +0200
+++ eccodes-1.1.0/setup.py	2021-01-23 20:08:50.936683323 +0100
@@ -45,6 +45,8 @@ setuptools.setup(
     license="Apache License Version 2.0",
     url="https://github.com/ecmwf/eccodes-python",
     packages=setuptools.find_packages(),
+    setup_requires=["cffi"],
+    cffi_modules=["builder.py:ffibuilder"],
     include_package_data=True,
     install_requires=[
         "attrs",
