--- eccodes-1.1.0.unchanged/docs/conf.py	2020-06-24 14:43:03.000000000 +0200
+++ eccodes-1.1.0/docs/conf.py	2021-01-23 20:08:57.362513852 +0100
@@ -32,7 +32,7 @@ source_suffix = ".rst"
 master_doc = "index"
 
 # General information about the project.
-project = u"eccodes-python"
+project = u"eccodes"
 copyright = u"2017-, European Centre for Medium-Range Weather Forecasts (ECMWF)."
 
 # The version info for the project you're documenting, acts as replacement
@@ -40,7 +40,7 @@ copyright = u"2017-, European Centre for
 # the built documents.
 #
 # The full version, including alpha/beta/rc tags.
-release = pkg_resources.get_distribution("eccodes-python").version
+release = pkg_resources.get_distribution("eccodes").version
 # The short X.Y version.
 version = ".".join(release.split(".")[:2])
 
