--- eccodes-1.3.2.p1p2/gribapi/bindings.py	2021-04-17 10:56:30.646189498 +0200
+++ eccodes-1.3.2.p1p2p3/gribapi/bindings.py	2021-04-17 10:56:56.435546620 +0200
@@ -24,14 +24,18 @@ __version__ = "1.3.2"
 
 LOG = logging.getLogger(__name__)
 
-try:
-    import ecmwflibs as findlibs
-except ImportError:
-    import findlibs
+#try:
+#    import ecmwflibs as findlibs
+#except ImportError:
+#    import findlibs
+#
+#library_path = findlibs.find("eccodes")
+#if library_path is None:
+#    raise RuntimeError("Cannot find the ecCodes library")
 
-library_path = findlibs.find("eccodes")
-if library_path is None:
-    raise RuntimeError("Cannot find the ecCodes library")
+import os
+import sys
+library_path = os.path.join(sys.prefix, sys.platlibdir, 'libeccodes.so')
 
 # default encoding for ecCodes strings
 ENC = "ascii"
--- eccodes-1.3.2.p1p2/setup.py	2021-04-17 10:56:30.646189498 +0200
+++ eccodes-1.3.2.p1p2p3/setup.py	2021-04-17 10:56:56.435546620 +0200
@@ -40,7 +40,7 @@ install_requires = ["numpy"]
 if sys.version_info < (3, 7):
     install_requires = ["numpy<1.20"]
 
-install_requires += ["attrs", "cffi", "findlibs"]
+install_requires += ["attrs", "cffi"] # , "findlibs"]
 
 setuptools.setup(
     name="eccodes",
--- eccodes-1.3.2.p1p2/eccodes.egg-info/requires.txt	2021-04-17 10:56:30.645189523 +0200
+++ eccodes-1.3.2.p1p2p3/eccodes.egg-info/requires.txt	2021-04-17 10:56:56.435546620 +0200
@@ -1,4 +1,3 @@
 numpy
 attrs
 cffi
-findlibs
